src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -157,9 +161,16 @@ def __init__(self, fuzz_target, backup_bucket_name, corpus_engine_name):
 class Context(object):
   """Pruning state."""
 
-  def __init__(self, fuzz_target, cross_pollinate_fuzzers):
+  def __init__(self,
+               fuzz_target,
+               cross_pollinate_fuzzers,
```

Comment:
Nit: I'm not sure how much this complicates the plumbing so feel free to ignore if it seems to add complexity, but could this and method be merged into one argument with a default of a new value that means "no cross pollination"?

Reply:
I misunderstood what this variable was tracking. Definitely seems fine to leave it as-is.